plugin/vcs/vcs.go

```diff
@@ -79,6 +79,15 @@ type UserInfo struct {
 	State UserState `json:"state"`
 }
 
+// ProjectMember is the API message for project member info.
+type ProjectMember struct {
+	Email           string    `json:"email"`
```

Comment:
The json field is the GitLab implementation which should stay inside the GitLab module.

While here is the vcs module, so it should have a more generate interface.

Otherwise, if we later add the GitHub support, we are likely to change this again.